translations/kubectl/it_IT/LC_MESSAGES/k8s.po

```diff
@@ -1617,7 +1617,7 @@ msgid ""
 msgstr ""
 "\n"
 "\t\tCrea un autoscaler che automaticamente sceglie e imposta il numero di pod "
-"che vengono eseguiti in un cluster di kubernets.\n"
+"che vengono eseguiti in un cluster di kubernetes.\n"
```

Comment:
should be uppercase `Kubernetes`.

Reply:
Thanks. I've changed all the words kubernetes in uppercase ,except those in url and file path.